experimental/babel-preset-env-standalone/test/.eslintrc

```diff
@@ -0,0 +1,8 @@
+{
+  "env": {
+    "mocha": true
```

Comment:
Might be better to just add it to:
https://github.com/babel/babel/blob/master/.eslintrc#L26